### PR TITLE
GM: Unify GM commands

### LINKED ISSUE #1019: Unify GM commands
##### SMALL DESCRIPTION:

Unify GM commands because some names are confusing

##### CURRENT BEHAVIOUR:

Most game objects commands are:

`.gobject`

But some are like that:

```
go object
list object
lookup object
```

I think the commands should be all using "gobject" instead of "object".
Maybe, there could be an alias for "object" in order to not break addons like TrinityGM. Or we could fix these addons :P

**There are probably several other commands in that case.**

##### BRANCH(ES):

master

##### AC HASH/COMMIT:

f66e0dc99e33ba3d4bcc1b2b2a689509bb50c816

### PR DESCRIPTION
**Pull request by Pondaveia**
##### CHANGES PROPOSED:

-  Added gobject to the list, look up and go commands in order to maintain a pattern.
-  Did not remove object commands to preserve Trinity add-ons compatibility.

###### ISSUES ADDRESSED:

Closes 
#1019 

##### TESTS PERFORMED:

- Compiles without errors and tested with success

##### HOW TO TEST THE CHANGES:

.go gobject 34
.list gobject
.lookup gobject x

##### Target branch(es):

Master

### CODE REVIEW

**Comment by BarbzYHOOL** [comment]
Hmm cool! Maybe you could also add to the help column of the deprecated commands `[DEPRECATED]: use go gobject` for example

**Comment by BarbzYHOOL** [comment]
so? no update?

File: src/server/scripts/Commands/cs_lookup.cpp [hunk 1]
```diff
@@ -43,21 +43,22 @@ class lookup_commandscript : public CommandScript
...
-            { "object",         SEC_MODERATOR,      true,  &HandleLookupObjectCommand,          "" },
-            { "quest",          SEC_MODERATOR,      true,  &HandleLookupQuestCommand,           "" },
-            { "player",         SEC_GAMEMASTER,     true,  nullptr,                                "", lookupPlayerCommandTable },
-            { "skill",          SEC_MODERATOR,      true,  &HandleLookupSkillCommand,           "" },
-            { "spell",          SEC_MODERATOR,      true,  nullptr,                                "", lookupSpellCommandTable },
-            { "taxinode",       SEC_MODERATOR,      true,  &HandleLookupTaxiNodeCommand,        "" },
-            { "tele",           SEC_MODERATOR,      true,  &HandleLookupTeleCommand,            "" },
-            { "title",          SEC_MODERATOR,      true,  &HandleLookupTitleCommand,           "" },
-            { "map",            SEC_MODERATOR,      true,  &HandleLookupMapCommand,             "" }
+            { "area",           SEC_MODERATOR,      true,  &HandleLookupAreaCommand,        "" },
```
Comment (BarbzYHOOL) [inline src/server/scripts/Commands/cs_lookup.cpp]: what happened here??

**Comment by BarbzYHOOL** [comment]
I updated the sql myself

File: src/server/scripts/Commands/cs_lookup.cpp (hunk 1 above)
Comment (Pondaveia) [reply src/server/scripts/Commands/cs_lookup.cpp]: I adjusted the tabs

**Comment by BarbzYHOOL** [comment]
Thank you @Pondaveia and @BarbzYHOOL :P

File: src/server/scripts/Commands/cs_go.cpp [hunk 2]
```diff
@@ -32,6 +32,7 @@ class go_commandscript : public CommandScript
             { "graveyard",      SEC_MODERATOR,      false, &HandleGoGraveyardCommand,         "" },
             { "grid",           SEC_MODERATOR,      false, &HandleGoGridCommand,              "" },
             { "object",         SEC_MODERATOR,      false, &HandleGoObjectCommand,            "" },
```
Comment (BarbzYHOOL) [inline src/server/scripts/Commands/cs_go.cpp]: Just wondering if we should've added a "deprecated" comment here
```suggestion
            { "object",         SEC_MODERATOR,      false, &HandleGoObjectCommand,            "" }, // DEPRECATED: use gobject instead
```

File: src/server/scripts/Commands/cs_list.cpp [hunk 3]
```diff
@@ -31,6 +31,7 @@ class list_commandscript : public CommandScript
             { "creature",       SEC_MODERATOR,  true,  &HandleListCreatureCommand,          "" },
             { "item",           SEC_MODERATOR,  true,  &HandleListItemCommand,              "" },
             { "object",         SEC_MODERATOR,  true,  &HandleListObjectCommand,            "" },
```
Comment (BarbzYHOOL) [inline src/server/scripts/Commands/cs_list.cpp]: Same here

**Pull request merged**